includes/lib/transifex-live-integration-hreflang.php

```diff
@@ -107,10 +108,13 @@ private function generate_languages_hreflang( $raw_url, $languages,
 				$href_link_parts[0] = $protocol;
 				$arr['href'] = implode(':', $href_link_parts);
 			} else {
-				$arr['href'] = $url_map[$language];
+        $arr['href'] = $url_map[$language];
 			}
 			$arr['hreflang'] = $hreflang_map[$language];
+      // source language is placed first in the array
+      $arr['is_source'] = $firstIteration ? true : false;
 			array_push( $ret, $arr );
+      $firstIteration = false;
```

Comment:
Just a suggestion, if we would like to avoid declaring variable $firstIteration we could have the following
```suggestion
     foreach ($languages as $Index => $language) {
     .......
     $arr['is_source'] = ($index === 0);
     .......
     }
```

NOTE: if someone has not updated the plugin yet and source language is not stored at settings, I think we would get the first target language as source